RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/CompileDialog.java

```diff
@@ -60,9 +60,11 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public Dialog onCreateDialog(Bundle savedInstanceState) {
-        if(!mAlreadyCompiled){
-            mListener.onPositiveClick(CompileDialog.this);
-        }
+        // NOTE: This is commented out because we don't want the compile to execute without the user
+        //    clicking "OK" on the dialog. If Joe agrees, he can delete this.
+        // if(!mAlreadyCompiled){
+        //     mListener.onPositiveClick(CompileDialog.this);
+        // }
         return new AlertDialog.Builder(getActivity())
             .setTitle("Warning")
             .setMessage("Re-compiling a chapter will over-write the current audio and reset the checking level.")
```

Comment:
We'll revert to the original plan of only showing the dialog on re-compile, not for the first time.